examples/event_handling/poly_editor.py

```diff
@@ -141,6 +141,10 @@ def motion_notify_callback(self, event):
         x, y = event.xdata, event.ydata
 
         self.poly.xy[self._ind] = x, y
+        if self._ind == 0:
+            self.poly.xy[-1] = x, y
+        elif self._ind == len(self.poly.xy) - 1:
```

Comment:
does it make senses to check if `if self._ind == -1`? 

I wouldn't think that it would ever come up, but I'm not good with the interactive stuff.